src/crimson/os/seastore/cache.h

```diff
@@ -292,6 +292,7 @@ class Cache {
   using get_extent_ret = get_extent_ertr::future<TCachedExtentRef<T>>;
   template <typename T, typename Func, typename OnCache>
   get_extent_ret<T> get_extent(
+    const Transaction::src_t t_src, ///< [in] transaction src type
     paddr_t offset,                ///< [in] starting addr
     seastore_off_t length,          ///< [in] length
     const src_ext_t* p_metric_key, ///< [in] cache query metric key
```

Comment:
Now I think we can reuse this parameter (probably rename to p_src_ext) because some callers do not belong to any transaction. p_src_ext will be a nullptr in this case.

Reply:
ok